README.md

```diff
@@ -1,15 +1,71 @@
 # GeoMakie
+
 ## Geographic plotting utilities for Makie.jl <a href = "https://www.github.com/JuliaPlots/Makie.jl"><img src="https://raw.githubusercontent.com/JuliaPlots/Makie.jl/master/assets/logo.png" alt="Makie.jl" height="30" align = "top"></a>
 
 [![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://JuliaPlots.github.io/GeoMakie.jl/stable)
 [![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://JuliaPlots.github.io/GeoMakie.jl/dev)
-[![Build Status](https://gitlab.com/JuliaGPU/GeoMakie-jl/badges/master/pipeline.svg)](https://gitlab.com/JuliaGPU/GeoMakie-jl/pipelines)
-
-## Installation
+![CI](https://github.com/JuliaPlots/GeoMakie.jl/actions/workflows/ci.yml/badge.svg)
 
-This package is **in development** and will **break often**.  You can install it from the REPL like so:
+This package is **in development**, and may break often.  You can install it from the REPL like so:
 ```julia
 ]add GeoMakie
```

Comment:
```suggestion
julia>]
pkg> add GeoMakie
```

Reply:
IMO this is a pretty common theme in READMEs to have `]add PackageName` with nothing else...plus it's easily copy-pasteable while the multiline version is not.

Reply:
sure. It's just that in my experience teaching Julia, I always need to repeat myself, specially at the beginning, that we need to go first into the package mode and then we can add a new package. That's it. The one liner is also ok, ;)